cves/kernel/CVE-2012-2745.yml

```diff
@@ -452,7 +440,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This is most likely a planning error or slip. This is because there is no mention in the vulnerability introducing commit about clearing the session keyring. Either it was overlooked in the planning phase or was something they missed when coding 
```

Comment:
Do you think this vulnerability was caused by a lack of competence or something so hard to catch you can't blame them?

Reply:
I think a bit of both, I think it's along the line where in C you know when you malloc, a free should come but it got forgotten.